changelog/unreleased/issue-4942

```diff
@@ -0,0 +1,10 @@
+Enhancement: create snapshot summary statistics for snapshots which don't have them
+
+When restic rewrite is used with the --snapshot-summary [-s] option,
+a new snapshot is created containing statistics summary data. Only two fields
+will be non-zero: TotalFilesProcessed and TotalBytesProcessed.
+
+There is a logic change in cmd_rewrite: statistics data will always be attached,
```

Comment:
`cmd_rewrite` is an internal filename. The changelog should be written from a user perspective that is use `rewrite` here.